app/es_indices/observation_index.rb

```diff
@@ -168,7 +168,17 @@ def as_indexed_json(options={})
           project_observations.select{ |po| po.curator_identification_id.nil? }.
             map(&:project_id)).compact.uniq,
         project_observations: (indexed_project_ids || project_observations).map{ |po|
-          { project_id: po[:project_id], user_id: po[:user_id], uuid: po[:uuid] } },
```

Comment:
We shouldn't touch this right now. Node will fetch project observation preferences dynamically from the DB, so you don't need to add anything to the index.

I'm currently working on adding preferences to the index, and lots of other stuff, but that's not ready right now. So for this fix you should continue to use the old way, which means you don't need to change anything on the Rails end